### PR TITLE
MAR-4055-admin-frontend-change-brief-to-opportunity

### PR DESCRIPTION
**Pull request by Revit**

### CODE REVIEW

File: app/templates/view_buyers.html [hunk 1]
```diff
@@ -55,7 +55,7 @@
 {% if brief %}
     <div>
         <hr/>
-        <h3>Brief Details</h3>
+        <h3>Opportunity Details</h3>
```
Comment (Reshma-M-A) [inline app/templates/view_buyers.html:58]: There are still some instances of briefs mentioned in this file. Check line 29 and 31 for example.

Comment (Revit) [reply app/templates/view_buyers.html:58]: fixed line 29 and 31 line

**Review by Reshma-M-A — COMMENTED** [review]
Found some more instances of briefs in `view_buyers.html`.
Check line 29 and 31

File: app/templates/view_team.html [hunk 2]
```diff
@@ -107,7 +107,7 @@ <h3>Team Members' Permissions</h3>
 </div>
 
 <div>
-  <h3>Team Briefs</h3>
+  <h3>Team Opportunites</h3>
```
[the remark this answers is not in the document]
Comment (Revit) [reply app/templates/view_team.html]: changed as suggested

**Review by tobiasbrown — APPROVED** [review]

**Pull request merged**